src/DotNetWorker.Core/DotNetWorker.Core.csproj

```diff
@@ -32,5 +32,6 @@
     <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="5.0.0" />
     <PackageReference Include="System.Threading.Tasks.Extensions" Version="4.5.4" />
     <PackageReference Include="Microsoft.Extensions.Options" Version="5.0.0" />
+    <PackageReference Include="System.Text.Encodings.Web" Version="8.0.0" />
```

Comment:
Did you mean to change this to 8X (than 6x)?

Reply:
Yes. 6.x will reach EOL very soon. Moving to 8.0.0 will avoid another major bump.